packages/automerge-repo/src/network/NetworkAdapter.ts

```diff
@@ -1,29 +1,24 @@
 /* c8 ignore start */
 
 import { EventEmitter } from "eventemitter3"
+import { NetworkAdapterEvents, PeerMetadata } from "../index.js"
 import { PeerId } from "../types.js"
 import { Message } from "./messages.js"
-import { StorageId } from "../storage/types.js"
-
-/**
- * Describes a peer intent to the system
- * storageId: the key for syncState to decide what the other peer already has
- * isEphemeral: to decide if we bother recording this peer's sync state
- *
- */
-export interface PeerMetadata {
-  storageId?: StorageId
-  isEphemeral?: boolean
-}
+import { NetworkAdapterInterface } from "./NetworkAdapterInterface.js"
 
 /** An interface representing some way to connect to other peers
+ * @deprecated
```

Comment:
I think this should be more like `@deprecated use {@link NetworkAdapterInterface} instead`. You can run `npx typedoc` to see what the output looks like.